access_request.go

```diff
@@ -7,6 +7,9 @@ type AccessRequest struct {
 	GrantTypes       Arguments `json:"grantTypes" gorethink:"grantTypes"`
 	HandledGrantType Arguments `json:"handledGrantType" gorethink:"handledGrantType"`
 
+	RefreshTokenRequestedScope Arguments
```

Comment:
This needs `json` tags IMO

Reply:
This has been removed as a discussion and code examples from Vivek highlighted it was not necessary.